AgentDiscoveries-Backend/src/main/resources/db/migration/V7__add_report_title_field.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE location_reports ADD report_title mediumtext NOT NULL;
```

Comment:
Is there a reason we've used `mediumtext` rather than `varchar(N)` for some sensible N?